src/dedicated/overview/monitoring.md

```diff
@@ -6,7 +6,7 @@ These alerts are sent to our support and operations teams, and are not directly
 
 ## Monitoring systems
 
-We use well known open source tooling to collect metrics and to alert our staff if any of these metrics goes out of bounds.  We use Munin for collecting time-series data on server metrics, and dashboarding of these metrics so that we can monitor trends over time.  We also use Nagios as a point in time alerting system for our operations staff.
+Platform.sh uses well-known open source tooling to collect metrics and to alert our staff if any of these metrics goes out of bounds.  That includes use Munin for collecting time-series data on server metrics, and dashboarding of these metrics so that our team can monitor trends over time.  It also includes use Nagios as a point in time alerting system for our operations staff.
```

Comment:
```suggestion
Platform.sh uses well-known open source tooling to collect metrics and to alert our staff if any of these metrics goes out of bounds.  That includes the use of Munin for collecting time-series data on server metrics, and dashboarding of these metrics so that our team can monitor trends over time.  It also includes use Nagios as a point in time alerting system for our operations staff.
```